### PR TITLE
Pre-load fonts added to header.html

### PR DESCRIPTION
**Pull request by jhonny218**
## Description

Pre-loading the 4 main fonts to improve FCP (First Contentful Paint)

## Why is this done

- Using pre-load in some of the fonts can improve the performance. 
- By doing this, it helps the FCP time since the webfonts will not be blocked by the CSS files.
- These 4 web-fonts are important because they are called in the majority of the pages.

## Testing done
Locally

## Screenshots

**BEFORE**

Screen Shot 2019-12-12 at 12 15 55 PM

**AFTER**

Screen Shot 2019-12-12 at 12 18 59 PM

## Definition of done
- [x ] FCP Improved

### CODE REVIEW

**Comment by jhonnyoddball** [comment]
My `Prettier` plugin in VS Code made a lot of rearranging  but no changes other than the Pre-load tags

**Review by rianfowler — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

File: src/site/includes/header.html [hunk 1]
```diff
@@ -3,80 +3,91 @@
...
 
-<meta name="msapplication-TileImage" content="/img/design/icons/apple-touch-icon-114x114">
-<meta name="msapplication-TileColor" content="#144073">
+  <meta name="msapplication-TileImage" content="/img/design/icons/apple-touch-icon-114x114">
+  <meta name="msapplication-TileColor" content="#144073">
 
-<!-- CSS -->
-<link rel="stylesheet" data-entry-name="style.css">
-<link rel="stylesheet" data-entry-name="{{ entryname | default: 'static-pages' }}.css">
+  <!-- Preload main fonts -->
```
Comment (rianfowler) [inline src/site/includes/header.html]: maybe mention something about why we're doing this

**Review by rianfowler — APPROVED** [review]

**Review by Clark-W-Griswold — APPROVED** [review]
👍

File: src/site/includes/header.html (hunk 1 above)
Comment (jhonnyoddball) [reply src/site/includes/header.html]: Done in the initial description.

**Pull request merged**